From 70d9084b18b61ae88376b8e6a11a4e733672a301 Mon Sep 17 00:00:00 2001
From: Ren <89468157+Shirayuki39@users.noreply.github.com>
Date: Tue, 24 Sep 2024 15:35:42 +0000
Subject: [PATCH] build.config.gki: Disable check_defconfig

* Bruh

Signed-off-by: Ren <89468157+Shirayuki39@users.noreply.github.com>
---
 build.config.gki | 1 -
 1 file changed, 1 deletion(-)

diff --git a/build.config.gki b/build.config.gki
index 44d4ed1bee1f..43ba6fc460ac 100644
--- a/build.config.gki
+++ b/build.config.gki
@@ -1,3 +1,2 @@
 DEFCONFIG=gki_defconfig
-POST_DEFCONFIG_CMDS="check_defconfig"
 
