From 6524b6ef533e5c850b3b6e56c27a8264a05005a5 Mon Sep 17 00:00:00 2001
From: negroweed <134845563+negroweed@users.noreply.github.com>
Date: Mon, 2 Dec 2024 22:12:13 +0800
Subject: [PATCH] setlocalversion: disable '-dirty' string in kernel release

---
 scripts/setlocalversion | 19 -------------------
 1 file changed, 19 deletions(-)

diff --git a/scripts/setlocalversion b/scripts/setlocalversion
index 842936656b846..26094113db880 100755
--- a/scripts/setlocalversion
+++ b/scripts/setlocalversion
@@ -107,19 +107,6 @@ scm_version()
 			printf -- '-svn%s' "$(git svn find-rev $head)"
 		fi
 
-		# Check for uncommitted changes.
-		# First, with git-status, but --no-optional-locks is only
-		# supported in git >= 2.14, so fall back to git-diff-index if
-		# it fails. Note that git-diff-index does not refresh the
-		# index, so it may give misleading results. See
-		# git-update-index(1), git-diff-index(1), and git-status(1).
-		if {
-			git --no-optional-locks status -uno --porcelain 2>/dev/null ||
-			git diff-index --name-only HEAD
-		} | grep -qvE '^(.. )?scripts/package'; then
-			printf '%s' -dirty
-		fi
-
 		# All done with git
 		return
 	fi
@@ -138,12 +125,6 @@ scm_version()
 			fi
 		fi
 
-		# Are there uncommitted changes?
-		# These are represented by + after the changeset id.
-		case "$hgid" in
-			*+|*+\ *) printf '%s' -dirty ;;
-		esac
-
 		# All done with mercurial
 		return
 	fi
